packages/react/src/components/UIShell/SideNav.js

```diff
@@ -40,6 +41,9 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
   const [expandedViaHoverState, setExpandedViaHoverState] =
     useState(defaultExpanded);
   const expanded = controlled ? expandedProp : expandedState;
+  const sideNavRef = useRef(null);
+  const refNav = useMergedRefs([sideNavRef, ref]);
```

Comment:
I think the name `navRef` makes more sense here, just a suggestion tho'
```suggestion
  const navRef = useMergedRefs([sideNavRef, ref]);
```